docs/src/sections/NavbarSection.js

```diff
@@ -41,12 +41,14 @@ export default function NavbarSection() {
 
       <h3><Anchor id="navbars-mobile-friendly">Responsive Navbars</Anchor></h3>
       <p>
-        To have a mobile friendly Navbar, Add a <code>Navbar.Toggle</code> to your Header and wrap your
-        Navs in a <code>Navbar.Collapse</code> component. The <code>Navbar</code> will automatically wire
-        the toggle and collapse together!
+        To have a mobile-friendly Navbar, add a <code>Navbar.Toggle</code> to your Header and wrap your
+        Navs in a <code>Navbar.Collapse</code> component. The <code>Navbar</code> will automatically combine
+        the toggle and collapse.
       </p>
       <p>
-        Set the <code>defaultExpanded</code> prop to make the Navbar start expanded. Set <code>collapseOnSelect</code> to make the Navbar collapse automatically when the user selects an item. You can also finely control the collapsing behavior by using the <code>expanded</code> and <code>onToggle</code> props.
+        Set the <code>defaultExpanded</code> prop to make the Navbar start expanded. Set <code>collapseOnSelect</code>
```

Comment:
err sorry I'm a little new to this, I was just trying to keep it consistent and since there wasn't a lot to this file, and most of the other paragraphs were wrapped, I figured this would be as well. I couldn't find a style guide or anything so this was just a leap of faith that I had taken. There's no need to merge if it's unnecessary!